.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules
 coverage
 bench/.results
 types/generated.d.ts
+
+# VS Code configuration
+.vscode
```

Comment:
Sure. I didn't want to do that initially because I have other repos that I want `.vscode` files checked in for, but maybe I can use `.git/info/exclude` instead.